### PR TITLE
Conversations

### PR DESCRIPTION
**Pull request by tibo**
@vdaubry not completely finished but ready if you want to take a first look at it.

todo:
- add link comment link + new comments in digest
- add the ability to create the first comment in when posting (+API)

### CODE REVIEW

File: db/migrate/20150718155400_create_comments.rb [hunk 1]
```diff
@@ -0,0 +1,11 @@
+class CreateComments < ActiveRecord::Migration
+  def change
+    create_table :comments do |t|
+      t.integer :user_id
+      t.text :content
+      t.integer :link_id
+
+      t.timestamps null: false
+    end
+  end
```
Comment (vdaubry) [inline db/migrate/20150718155400_create_comments.rb:13]: We need to index comments by foreign keys :
add_index :comments, :link_id
add_index :comments, :user_id

File: db/migrate/20150718155400_create_comments.rb [hunk 2]
```diff
@@ -0,0 +1,11 @@
+class CreateComments < ActiveRecord::Migration
+  def change
+    create_table :comments do |t|
+      t.integer :user_id
```
Comment (vdaubry) [inline db/migrate/20150718155400_create_comments.rb]: we should not allow nil values for any fields :

:user_id, null: false
:link_id, null: false
:content, null: false

**Comment by tibo** [comment]
@vdaubry done.
should be good to go.
I guess we can make an other PR for all the changes related to the daily digest? (+ some fixes)

**Comment by vdaubry** [comment]
looks good, i'll do some testing before merging

**Comment by vdaubry** [comment]
Ok it works, but we'll need to improve the general look of conversations later

**Pull request merged**